File: Repl.it/C++/Switch.cpp
#include <iostream>
using namespace std;
int main()
{
    cout << "Введите два числа: ";
    float a, b, c;
    cin >> a >> b;
    cout << "Введите алгебраический знак: ";
    char operand;
    cin >> operand;
    if (operand == '+' || operand == '-' || operand == '*' || operand == '/')
        switch (operand)
        {
        case '+':
            c = a + b;
            cout << a << "+" << b << "=" << c;
            break;
        case '-':
            c = a - b;
            cout << a << "-" << b << "=" << c;
            break;
        case '*':
            c = a * b;
            cout << a << "*" << b << "=" << c;
            break;
        case '/':
            if (b == 0)
                cout << "Деление не возможно";
            else
            {
                c = a / b;
                cout << a << "/" << b << "=" << c;
            }
            break;
        }
    else
        cout << "Не шали!!";
}

File: Repl.it/C#/Switch.cs
using System;
class MainClass
{
    public static void Main(string[] args)
    {
        Console.WriteLine("Введите два числа: ");
        float a, b, c;
        a = Convert.ToInt32(Console.ReadLine());
        b = Convert.ToInt32(Console.ReadLine());
        char operand;
        Console.WriteLine("Введите знак: ");
        operand = Convert.ToChar(Console.ReadLine());
        switch (operand)
        {
            case '+':
                c = a + b;
                Console.WriteLine($"{a} + {b} = {a + b}");
                break;
            case '-':
                c = a - b;
                Console.WriteLine($"{a} - {b} = {a - b}");
                break;
            case '*':
                c = a * b;
                Console.WriteLine($"{a} * {b} = {a * b}");
                break;
            case '/':
                if (b == 0)
                    Console.WriteLine("Деление невозможно.");
                else
                {
                    c = a / b;
                    Console.WriteLine($"{a} / {b} = {a / b}");
                }
                break;
        }
    }
}